src/test/java/dk/kb/present/PresentFacadeTest.java

```diff
@@ -160,17 +180,14 @@ void getRecordsJSONLD() throws IOException {
     }
      */
 
-    /*  TODO FIX!
-    /   Can only be fixed, when the updated XSLT to JSON-LD has been reviewed and merged to master
     @Test
     void getRecordsJSONLDLines() throws IOException {
-        StreamingOutput out = PresentFacade.getRecords(null, "dsfl", 0L, -1L, "json-ld-lines");
+        StreamingOutput out = PresentFacade.getRecords(null, "dsfl", 0L, -1L, "json-ld-lines", ids -> ids);
         String result = toString(out);
         assertTrue(result.contains("\"@value\":\"Letters to\\/from David Simonsen\"}"), "Result should contain the name David Simonsen in the expected wrapping");
         assertFalse(result.contains(",\n"), "Result should not contain a comma followed by newline as it should be a multi-entry JSON-Lines");
         assertFalse(result.endsWith("]\n"), "Result should not end with ']' as it should be in JSON-Lin es");
     }
-*/
```

Comment:
This test is failing. I have no clue on what the comment _Can only be fixed, when the updated XSLT to JSON-LD has been reviewed and merged to master_ refers to. I believe it is REALLY old and isn't related to the XSLTs i've written.

Reply:
Removed